docs/marketplace/dev/back-end/system-requirements.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We already have such a doc https://spryker-docs.herokuapp.com/docs/marketplace/dev/setup/system-requirements.html

System requirements are very important, that is why I suggest leaving the existing doc in the sidebar and removing this file.